tests/Paket.Bootstrapper.Tests/Paket.Bootstrapper.Tests.csproj

```diff
@@ -265,7 +300,7 @@
     <When Condition="($(TargetFrameworkIdentifier) == '.NETStandard' And ($(TargetFrameworkVersion) == 'v1.3' Or $(TargetFrameworkVersion) == 'v1.4' Or $(TargetFrameworkVersion) == 'v1.5' Or $(TargetFrameworkVersion) == 'v1.6' Or $(TargetFrameworkVersion) == 'v2.0')) Or ($(TargetFrameworkIdentifier) == '.NETCoreApp' And ($(TargetFrameworkVersion) == 'v1.0' Or $(TargetFrameworkVersion) == 'v1.1' Or $(TargetFrameworkVersion) == 'v2.0'))">
       <ItemGroup>
         <Reference Include="System.Collections.NonGeneric">
-          <HintPath>..\..\packages\test\System.Collections.NonGeneric\ref\netstandard1.3\System.Collections.NonGeneric.dll</HintPath>
+          <HintPath>..\..\packages\test\System.Collections.NonGeneric.4.3.0\ref\netstandard1.3\System.Collections.NonGeneric.dll</HintPath>
```

Comment:
To elaborate: I don't think it makes sense blocking this, PR but we should open an issue to track that one down then.

Thanks for sending this standalone!